chart/compass/charts/director/templates/oauth-clients-scopes-sync-job.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
This job will probably rarely do any actual syncs (probably most of the times it would be just running without doing much). One idea for optimization would be that we annotate the job with the hash of the configmap its going to run with. And the next time we are about to run it we can check if the configmap hash has chaged and if not we skip running it.


Edit: I looked for some resources how this can be achieved, something similar for deployments is discussed in https://github.com/helm/helm/issues/2639 

One idea can be whats done above with the deployment in the above link. 

Another idea/variation of the one above could be that we look for the configmap, calculate the hash, look for the job and get the hashed annotation. Then we guard the job.yaml with an if template to only be created if the hash of the configmap is different from the hash of the job. lookups in charts we already do here https://github.com/kyma-incubator/compass/pull/1552/files

Reply:
We went with the second approach, thanks for the suggestion! The first one was not sufficient, because of the helm hooks we use in order to run after the migration job